lib/crypto.js

```diff
@@ -592,23 +592,21 @@ exports.Certificate = Certificate;
 function Certificate() {
   if (!(this instanceof Certificate))
     return new Certificate();
-
-  this._handle = new binding.Certificate();
```

Comment:
@bnoordhuis / @jasnell This is tagged as `semver-minor` but that seems incorrect; this looks like a major?